### ISSUE TITLE
Entering the login page initializes i18next config

### ISSUE THREAD

**Issue by jongwan56** [SoftwareBrothers/admin-bro#789]
**Describe the bug**
AdminBro calls i18n.init when somebody enters to the login page, so i18next config is overwritten.

**Installed libraries and their versions**
 * @admin-bro/express@3.1.0
 * admin-bro@3.4.0
 * express@4.17.1
 * express-formidable@1.2.0
 * express-session@1.17.1
 * i18next@19.9.2
 * tslib@2.1.0

**To Reproduce**
1. Run server (https://github.com/jongwan56/admin-bro-i18next-bug)
2. See the response of http://localhost:3000
3. Move to http://localhost:3000/admin/login
4. See the response of http://localhost:3000 again

**Expected behavior**
The response should be "안녕, 세상!" always.

**Additional context**
The problem is [here](https://github.com/SoftwareBrothers/admin-bro/blob/master/src/frontend/login-template.tsx#L56). 
I'm not sure why this code exists.

**Comment by dziraf** [comment]
This is an issue with `i18next` library which introduced a bug/breaking change in version `19.8.8`. AdminBro package uses:
```
    "i18next": "^19.1.0",
```
so it installs the most recent v19 of `i18next` whenever you run `npm/yarn install`.
In `admin-bro@v3.4.0-beta.8` (https://github.com/SoftwareBrothers/admin-bro/releases/tag/v3.4.0-beta.8) I added a temporary fix which sets `i18next` to use a fixed version `19.8.7`.

This issue is presumably fixed in version `19.9.2` of i18next but for now I've put the update on hold.

TL;DR:
Install `admin-bro@beta`.

**Comment by jongwan56** [comment]
It works! Thank you.
